doc/patching-3rdparties.md

```diff
@@ -1,6 +1,13 @@
 # Patching 3rd party dependencies
 
-If 3rd party dependency has an issue and fix is not yet released (or we can't switch to a new release), local patch can be applied. 
-Status doesn't used nodejs-way of applying patches. Instead we use Nix.
+## Libraries
+If 3rd party library has an issue and fix is not yet released (or we can't switch to a new release), we use forks. Fix should be commited to the fork and tagged.
+
+Example: [`react-native-hole-view`](https://github.com/status-im/react-native-hole-view#refs/tags/v2.1.1-status)
+
+## React Native
+When patch need to be applied to React Native itself Status does patching with Nix instead of doing it nodejs-way.
```

Comment:
what is nodejs-way ?

Reply:
https://www.npmjs.com/package/patch-package

Reply:
> what is nodejs-way ?
https://www.npmjs.com/package/patch-package

Reply:
Things like this - https://www.npmjs.com/package/patch-package